### PR TITLE
added option to change S3Uploader for FakeS3 when running with SECOR_…

### PR DESCRIPTION
**Pull request by nikshepsvn**
…DEV='true'

### CODE REVIEW

File: src/main/java/com/pinterest/secor/uploader/S3UploadManager.java [hunk 1]
```diff
@@ -128,6 +128,9 @@ public void refresh() {}
         if (mConfig.getAwsClientPathStyleAccess()) {
             S3ClientOptions clientOptions = new S3ClientOptions();
             clientOptions.setPathStyleAccess(true);
+            if (System.getenv("SECOR_DEV") == "true"){
+                clientOptions.disableChunkedEncoding();
```
Comment (HenryCaiHaiying) [inline src/main/java/com/pinterest/secor/uploader/S3UploadManager.java]: What was the problem with checked encoding?

Comment (HenryCaiHaiying) [reply src/main/java/com/pinterest/secor/uploader/S3UploadManager.java]: And when is SECOR_DEV being set?

Comment (drrzmr) [reply src/main/java/com/pinterest/secor/uploader/S3UploadManager.java]: I guess that @nikshepsvn are setting `SECOR_DEV` only in his development environment. IMHO this approach are like "easter eggs", better approach are use properties to change this, where it must be well documented what is the propose of this setting.

**Comment by nikshepsvn** [comment]
Oops everyone, was supposed to make this PR on a fork of this repo! What I've heard is that fakeS3 bugs out when using chunkedEncoding (see: https://github.com/jubos/fake-s3/issues/164) but even after I figured that out and got it working there was some SSL issues connecting to the bucket which I have created issues for.

**Pull request closed without merging**